### PR TITLE
JBPM-6876: Stunner - easy way to migrate from Stunner to current Designer

### PR DESCRIPTION
**Pull request by wmedvede**
- Adds the ability to optionally show an inline notification in the ConfirmPopup according to UX requeriment.

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @romartin @hasys would you mind review this PR?
Thanks.

File: uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/Button.java [hunk 1]
```diff
@@ -50,6 +59,8 @@ public void setType(final ButtonType type) {
     }
 
     public void setButtonStyleType(final ButtonStyleType type) {
+        removeClass("btn");
```
Comment (jomarko) [inline uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/Button.java]: Just question, why we remove this "btn" class when changing button style, but we declare this class by default, for example in `ConfirmPopup.html`:
```html
<button type="button" class="btn btn-default" data-field="confirm-cancel" data-dismiss="modal"></button>
<button type="button" class="btn btn-danger" data-field="confirm-ok"></button>
```

File: uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/ConfirmPopup.java [hunk 2]
```diff
@@ -36,27 +39,70 @@
...
+    @PostConstruct
+    public void init() {
+        cancelButton.setText(translationService.getTranslation(Constants.ConfirmPopup_Cancel));
+    }
+
     public void show(String title,
                      String okButtonText,
                      String confirmMessage,
                      final Command okCommand) {
+        show(title,
```
Comment (jomarko) [inline uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/ConfirmPopup.java]: Minor thing, could we please use `final` keyword for arguments of both `show` methods. I think it is good to be consistent with style of declaration `@EventHandler` methods in this class.

File: uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/Button.java (hunk 1 above)
Comment (wmedvede) [reply uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/Button.java]: The "btn" class is also removed for safety and to be sure that all the elements that configure the button style are properly set to the new style.

**Pull request merged**